src/EditorFeatures/Test2/PasteTracking/PasteTrackingServiceTests.vb

```diff
@@ -284,11 +218,11 @@ $$
 
                 testState.SendPaste(class1Document, PastedCode)
 
+                Dim textBuffer = class1LinkedDocument.GetTextBuffer()
+
                 testState.CloseDocument(class1Document)
                 testState.CloseDocument(class1LinkedDocument)
 
-                Await testState.AssertMissingPastedTextSpanAsync(class1LinkedDocument)
```

Comment:
Do we no longer need to assert here?

Reply:
It wasn't correct: the tests were testing that if the documents are closed, that the text buffer properties are nulled out. But the product doesn't do that -- the test harness itself was. It was testing the test harness rather than the actual product. Once we fixed the test harness by deleting code it started failing.

So what about the product? Yes, if you close the documents but the ITextBuffer is leaked, we'll also leak span information. If we want to fix that we can, but that's orthogonal to this, and I'm not sure that's really worth fixing.